markdown/ddl/ddl-database.html.md.erb

```diff
@@ -2,77 +2,75 @@
+You can also use the client program `createdb` to create a database. For example, running the following command connects to HAWQ on the host named `hmaster` at port `5432` and creates a database named `mydatabase`:
 
 ``` shell
-$ createdb -h masterhost -p 5432 mydatabase
+$ createdb -h hmaster -p 5432 mydatabase
 ```
 
-The host name and port must match the host name and port of the installed HAWQ system.
+The host name and port must match the host name and port of the HAWQ master node.
 
-Some objects, such as roles, are shared by all the databases in a HAWQ system. Other objects, such as tables that you create, are known only in the database in which you create them.
+Some objects, such as roles, are shared by all of the databases in a HAWQ deployment. Other objects, such as tables that you create, are known only to the database in which you create them.
 
 ### <a id="topic5"></a>Cloning a Database 
 
-By default, a new database is created by cloning the standard system database template, *template1*. Any database can be used as a template when creating a new database, thereby providing the capability to 'clone' or copy an existing database and all objects and data within that database. For example:
+By default, a new database is created by cloning the standard system database template, `template1`. You may identify an alternate template to use for database creation. As mentioned in the section above, you can specify `template0` as the template:
 
 ``` sql
-=> CREATE DATABASE new_dbname TEMPLATE old_dbname
+=# CREATE DATABASE <new_dbname> WITH TEMPLATE template0;
 ```
 
 ## <a id="topic6"></a>Viewing the List of Databases 
 
-If you are working in the `psql` client program, you can use the `\l` meta-command to show the list of databases and templates in your HAWQ system. If using another client program and you are a superuser, you can query the list of databases from the `pg_database` system catalog table. For example:
+You can use the `\l` meta-command of the `psql` client subsystem to display the list of databases and templates in your HAWQ deployment. If you are a database superuser, you can query the list of databases from the `pg_database` system catalog table. For example:
 
 ``` sql
-=> SELECT datname FROM pg_database;
+=# SELECT datname FROM pg_database;
 ```
 
 ## <a id="topic7"></a>Altering a Database 
 
-The ALTER DATABASE command changes database attributes such as owner, name, or default configuration attributes. For example, the following command alters a database by setting its default schema search path \(the `search_path` configuration parameter\):
+Use the [ALTER DATABASE](../reference/sql/ALTER-DATABASE.html) command to update database attributes including owner or name. You can also use ALTER DATABASE to update a database's default configuration. For example, the following command alters a database by setting its default schema search path using the `search_path` server configuration parameter:
```

Comment:
"For example, the following command alters a database by setting its default schema search path using the `search_path` server configuration parameter:" -> "For example, the following command alters the `search_path` server configuration parameter to change the default schema search path:"